liquibase-core/src/main/java/liquibase/datatype/core/TimestampType.java

```diff
@@ -130,7 +143,7 @@ public DatabaseDataType toDatabaseDataType(Database database) {
 
             if (additionalInformation != null) {
                 String additionInformation = additionalInformation.toUpperCase(Locale.US);
-                if ((database instanceof PostgresDatabase) && additionInformation.toUpperCase(Locale.US).contains("TIMEZONE")) {
+                if ((database instanceof PostgresDatabase || database instanceof H2Database) && additionInformation.toUpperCase(Locale.US).contains("TIMEZONE")) {
                     additionalInformation = additionInformation.toUpperCase(Locale.US).replace("TIMEZONE", "TIME ZONE");
                 }
```

Comment:
I don't understand where `TIMEZONE` may came from, but other databases don't support `TIMEZONE` too, not only the PostgreSQL and H2.

Reply:
If it comes from the code above, it should be adjusted instead to produce a correct SQL. But maybe there is some other source of incorrect name? It that case this fix should be unconditional for all databases.